src/infrastructure/utils/parseDate.ts

```diff
@@ -0,0 +1,19 @@
+export function parseDate(date: Date): string {
+  // Массив с названиями месяцев
+  const months = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'];
```

Comment:
we can try using native Intl module https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat